### DISCUSSION TITLE
已解决，打扰了。

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#117] [Q&A] [answered]
我在试图将一个char*转为string时发生了意料之外的错误，经断点调试，char*内容正常，但string报错，上网搜寻无果，希望得到帮助。
![image](https://user-images.githubusercontent.com/100075314/157205604-5ec4e897-d692-4360-851c-9324dade88e2.png)
![image](https://user-images.githubusercontent.com/100075314/157205694-1250fc4c-424a-4f74-9322-c8bd88e3d3a5.png)

**Comment by RicoloveFeng** [comment, marked as answer]
.
